scroll depth (#573)
plausible/docs: docs/scroll-depth.md

This data provides insights into user engagement and helps optimize content placement, calls-to-action (CTAs) and page layouts.

## Scroll Depth percentage for your pages

You can find scroll depth data in:

1.  **The top row of metrics** when a page filter is applied. Click on it to see trends over time.

2.  **The expanded Top Pages report**, where pages can be sorted by scroll depth for deeper analysis.

<img alt="Scroll depth metric in Plausible" src={useBaseUrl('img/scroll-depth.png')} />

If there's insufficient data, scroll depth will display as "-" until more traffic is recorded.

You can also use [the Filters feature](filters-segments.md) to group similar pages and analyze their average scroll depth. For example, filter URLs containing "blog" to see average engagement for blog posts.

Plausible tracks scrolling at all percentages (1% to 100%), unlike many other tools that limit tracking to specific points such as 25%, 50%, and 75%. If you also want to track a specific scroll depth threshold for some of your pages, you can set up scroll depth goals.

> ## Scroll Depth goals

Reviewer: There are two limitations that I think we should mention in this section.

1. Number of conversions might appear lower than it really is

> Our tracking script captures scroll depth at the moment when the visitor **leaves a page**. But it can't always do so due to browser limitations. E.g. sometimes, browsers won’t detect when a user leaves if they switch apps or move the browser to another screen. While we use best practices to maximize accuracy, some of these events will always be missed.

2. Number of total conversions is not available for Scroll Depth goals

> While for pageview and custom event goals we can easily tell how many events there were in total, we cannot say the same for Scroll Depth goals as scrolling is an action that gets measured continuously during a pageview.

Reviewer 2: I have added the second point, @metmarkosaric . If it looks good (you should be able to see it below), I will commit the suggestion.

For the accuracy part (first point), I am thinking it is something we don't need to say as it could overcomplicate and unnecessarily raise questions about our accuracy and increase the support volume as well.

Reviewer: Commited the suggestion below. Thanks @Hricha-Shandily!

About the first point though, I do believe it's important to note this in the docs. We could perhaps phrase it a bit differently, but it's a significant factor affecting an important number on the dashboard, so I feel like it should definitely be added into the docs. And the "Scroll Depth Goals" section in the `scroll_depth.md` document sounds like the right place for it.

That said, it's also fine by me if we roll it out like that and see how people will respond first.

Click on the "**+ Add goal**" button to go to the goal creation form.

Select `Scroll Depth` as the goal trigger, select the scroll depth percentage threshold (from 1% to 100%) and enter the pathname of the page you would like to track. The pathname must match the page path you can see in your Plausible Analytics dashboard.

Do you want to analyze the scroll depth for a specific section of your site? You can use an asterisk (`*`) to match patterns in your page path URLs. Asterisks can be placed on either end or in the middle of any page path URL.

You can for instance group all of your blog posts by using `/blog*` (if your blog subdirectory is named `blog`) or your Woocommerce checkout pages by using `/checkout/order-received/*`. 

Next, click on the "**Add goal**" button to return to the goals page. When you navigate back to your Plausible Analytics dashboard, you should see the number of visitors who reached the particular scroll depth on the specified pages. Goal conversions are listed at the bottom of the dashboard.

> 

Reviewer 2: ```suggestion

Scroll Depth goals only display the "Uniques" and "CR (conversion rate)" metrics, and not the "Total" metric like the pageview goals and custom event goals do. This is because scrolling is an action that gets measured continuously during a pageview.

```

> Next, click on the "**Add goal**" button and you’ll be taken back to the goals page. When you navigate back to your Plausible Analytics dashboard, you should see the number of visitors who have reached the particular scroll depth on the specified pages. Goal conversions are listed at the very bottom of the dashboard.

Reviewer 2: ```suggestion
Next, click on the "**Add goal**" button to return to the goals page. When you navigate back to your Plausible Analytics dashboard, you should see the number of visitors who reached the particular scroll depth on the specified pages. Goal conversions are listed at the bottom of the dashboard.
```
